internal/cmd/apit/setsnippet.go

```diff
@@ -0,0 +1,92 @@
+package apit
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/saucelabs/saucectl/internal/apitest"
+	"github.com/saucelabs/saucectl/internal/http"
+	"github.com/spf13/cobra"
+)
+
+func SetSnippetCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set-snippet NAME FILE_NAME|- [--project PROJECT_NAME]",
+		Short: "Set a vault snippet",
+		Long: `
+Set/update a snippet in a project's vault. If a snippet NAME is already in the vault,
+the value will be updated otherwise a new snippet will be added. You can set a snippet's
```

Comment:
I'm not a grammar expert, but it needs a comma somewhere 🔍 
```suggestion
the value will be updated, otherwise a new snippet will be added. You can set a snippet's
```